documentation/modules/con-securing-client-authentication.adoc

```diff
@@ -5,21 +5,33 @@
 [id='con-securing-client-authentication-{context}']
 = User authentication
 
+[role="_abstract"]
 User authentication is configured using the `authentication` property in `KafkaUser.spec`.
 The authentication mechanism enabled for the user is specified using the `type` field.
 
-Supported authentication mechanisms:
+Supported authentication types:
 
-* TLS client authentication
-* SCRAM-SHA-512 authentication
+* `tls` for TLS client authentication
+* `tls-external` for TLS client authentication using external certificates
+* `scram-sha-512` for SCRAM-SHA-512 authentication
 
-When no authentication mechanism is specified, the User Operator does not create the user or its credentials.
+If `tls` or `scram-sha-512` is specified, the User Operator creates authentication credentials when it creates the user.
+If `tls-external` is specified, a TLS user is created without authentication credentials.
+When no authentication type is specified, the User Operator does not create the user or its credentials.
 
+Use `tls-external` to authenticate with TLS using a certificate issued outside the User Operator.
+The User Operator creates a TLS user, but it does not generate a TLS certificate or a secret.
```

Comment:
Same as above, it does not really create the user because the user is created just by creating the certificate.

Reply:
Changed to: _The User Operator does not generate a TLS certificate or a secret._